src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -975,13 +979,19 @@ private static void tryAddWinningBidByBidder(Bid bid, String bidder,
             result = cacheService.cacheBidsOpenrtb(bidsWithNonZeroCpm, imps, CacheContext.of(
                     cacheInfo.shouldCacheBids, cacheInfo.cacheBidsTtl, cacheInfo.shouldCacheVideoBids,
                     cacheInfo.cacheVideoBidsTtl), publisherId, timeout)
-                    .recover(throwable -> Future.succeededFuture(Collections.emptyMap())) // skip cache errors
+                    .recover(throwable -> logCacheErrorAndReturnEmptyList(throwable, errors))
                     .map(bidToCacheId -> addNotCachedBids(bidToCacheId, bids));
         }
 
         return result;
     }
 
+    private Future<Map<Bid, CacheIdInfo>> logCacheErrorAndReturnEmptyList(Throwable throwable, List<String> errors) {
+        errors.add("Error occurred while trying to cache : " + throwable.getMessage());
```

Comment:
Please change wording to something along the lines of "Error occurred while trying to cache bids. Message: "